packages/tables/src/Table/Concerns/CanSortRecords.php

```diff
@@ -24,6 +24,8 @@ public function defaultSort(string | Closure | null $column, string | Closure |
             $this->defaultSortColumn = $column;
         }
 
+        $direction = $this->evaluate($direction);
```

Comment:
Evaluation should always happen in getters, it should not be evaluated here.

Reply:
ive changed it

Reply:
> Evaluation should always happen in getters, it should not be evaluated here.

I had done that first, as you may have seen.  I noticed that this was normally in the getters.  But, I changed my mind thinking you might prefer to keep the changes smaller.

Reply:
> ive changed it

Thx.  You did what I had done on my first iteration.  Makes me feel good that I had the right solution...before I changed it.  😅